src/components/ContextBar.js

```diff
@@ -117,11 +137,34 @@ export const ContextBar = ({
         await refreshRuntimes(true)
         await refreshApps()
       },
-      runtimes, apps, appDataDisks, refreshRuntimes, refreshApps, workspace, canCompute, persistentDisks, location, locationType
+      runtimes, apps, appDataDisks, refreshRuntimes, refreshApps, workspace, canCompute, persistentDisks, location, computeRegion
     }),
     div({ style: { ...Style.elements.contextBarContainer, width: 70 } }, [
       div({ style: contextBarStyles.contextBarContainer }, [
         h(Fragment, [
+          h(Clickable, {
```

Comment:
I don't know if clickable is the right element to use here. This results in it having a hover-over color, and the cursor changing into a `click here` affordance. Was a bit surprised nothing happened when I clicked based on this. A div or the like may be more applicable

![image](https://user-images.githubusercontent.com/6465084/179800625-6b7c6def-7af5-4056-92c0-756d999703cc.png)